diff --git a/Makefile b/Makefile
index c130ed6..42609e7 100644
--- a/Makefile
+++ b/Makefile
@@ -1,12 +1,12 @@
-CC=gcc
+CC=i586-mingw32msvc-gcc
 CFLAGS=\
 	-Dstricmp=strcasecmp -DCom_Memcpy=memcpy -DCom_Memset=memset \
-	-DMAC_STATIC= -DQDECL= -DLINUX -DBSPC -D_FORTIFY_SOURCE=2 \
+	-DMAC_STATIC= -DQDECL= -DWIN32 -DBSPC -D_FORTIFY_SOURCE=2 \
 	-I. -Ideps -Wall
 
 RELEASE_CFLAGS=-O3 -ffast-math
 DEBUG_CFLAGS=-g -O0 -ffast-math
-LDFLAGS=-lm -lpthread
+LDFLAGS=-lm
 
 DO_CC=$(CC) $(CFLAGS) -o $@ -c $<
 
